en/mozorg/about/governance/policies/reporting.ftl

```diff
@@ -0,0 +1,52 @@
+reporting-if-you-have-a-report-involving = If you have a report <strong>involving an employee, contractor, or vendor</strong> (for example, the report is made <strong>by</strong> an employee or is <strong>about</strong> an employee) then you should report at the <a href="{ $employee_hotline }">{ -brand-name-mozilla } Employee hotline</a>.
+reporting-put-another-way = Put another way…
+reporting-by = By
+reporting-employee = Employee
+reporting-contributor = Contributor
+reporting-about = About
+reporting-employee-hotline = Employee Hotline
+reporting-community-hotline = Community Hotline
+reporting-contractor = Contractor
+reporting-vendor = Vendor
+reporting-if-someone-reports-to = If someone reports to you…
+reporting-do-not-question-or-judge = Do not question, or judge their experience.
+reporting-do-not-invite-them-to = Do not invite them to withdraw the incident report.
+reporting-do-not-promise-any-particular = Do not promise any particular response.
+reporting-do-let-them-know-that = <strong>Do</strong> let them know that for { -brand-name-mozilla }’s policy to be impactful, reports should go through the hotline. If they do not feel comfortable filing the report themselves, you may do so.
+reporting-no-matter-who-files-the = No matter who files the report, the information below is important to capture.
+reporting-names-of-the-people-involved = Names of the people involved (or if names are unknown, use descriptions and any identifiable info such as appearance, role, handle, project/community affiliation).
+reporting-description-of-incident = Description of incident, including memorable dates (or event) and locations.
+reporting-if-the-reporter-wants = If the reporter wants to make an anonymous report, please inform them that this contact information we may not be able to update the initial reporter if appropriate. Some laws prohibit anonymous reporting and that you may be required to provide their name if you are a { -brand-name-mozilla } Manager or Community Leader.
+reporting-relationship-of-reportervictim = Relationship of reporter/victim.
+reporting-mozilla-managers-and-community = { -brand-name-mozilla } Managers and Community leaders
+reporting-if-a-mozilla-manager-or = If a { -brand-name-mozilla } Manager or Community leaders is informed about potential <a href="{ $cpg }">CPG</a> violations they are expected to immediately report the incident through the applicable hotline, even if the initial reporter will also file a report. { -brand-name-mozilla } Managers and Community leaders are not permitted to investigate complaints on their own.
+reporting-do-not-impose-your-own = Do not impose your own judgement on how the reporter should react. { -brand-name-focus } on listening.
```

Comment:
- `Focus` here is a verb, not a placeholder
- This string appears to be bold in the staging. I don't see any markup to indicate so.